libqtile/widget/check_updates.py

```diff
@@ -36,7 +36,8 @@ class CheckUpdates(base.ThreadedPollText):
         ("display_format", "Updates: {updates}", "Display format if updates available"),
         ("colour_no_updates", "ffffff", "Colour when there's no updates."),
         ("colour_have_updates", "ffffff", "Colour when there are updates."),
-        ("restart_indicator", "", "Indicator to represent reboot is required. (Ubuntu only)")
+        ("restart_indicator", "", "Indicator to represent reboot is required. (Ubuntu only)"),
+        ("refresh_after_execute", True, "Immediately refresh the widget after executing updates.")
```

Comment:
I thought about this for a bit. And do we *really* expect somebody to set `refresh_after_execute` to `False` because he doesn't want to see the widget update right after a system update? I can't see a realistic use-case for it.

@tych0 : What do you say? You seem to be in favor of adding options most of the time, so you probably have a better perspective. :)

Reply:
I agree that there's no reason to add an option here.